packages/suite/src/components/wallet/TransactionItem/components/Target/index.tsx

```diff
@@ -39,16 +39,18 @@ export const TokenTransfer = ({
     ...baseLayoutProps
 }: TokenTransferProps) => {
     const operation = getTxOperation(transfer);
+
     return (
         <BaseTargetLayout
             {...baseLayoutProps}
             addressLabel={<TokenTransferAddressLabel transfer={transfer} type={transaction.type} />}
             amount={
                 !baseLayoutProps.singleRowLayout && (
-                    <StyledHiddenPlaceholder>
-                        {operation && <Sign value={operation} />}
-                        {transfer.amount} {transfer.symbol}
-                    </StyledHiddenPlaceholder>
+                    <StyledCryptoAmount
+                        value={transfer.amount}
+                        symbol={transfer.symbol as NetworkSymbol}
```

Comment:
could be token symbol, so general `string`, not only `NetworkSymbol`

Reply:
5680bf19232d515b4216cde06def4f2d81f5297e